Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -14452,6 +14452,14 @@ end
 				{
 					Notes = "Damage from contact with a cactus block"
 				},
+				dtMagmaContact =
```

Comment:
You may wish to `git rebase -i origin/master` and delete the extra commits :)

Reply:
Will do!